buildo/nemobot
# #110: enforce single-assignee per issue/PR (closes #110)

Issue #110

## Test Plan

### tests performed
tested on https://github.com/buildo/prisma/issues/33

- add assignee
- add another assignee
  - nemobot removes first assignee (but... you have to refresh the page to see the change 😞 )
- clear assignees
- add multiple assignees in single action
  - nemobot removes **every** assignee. This happens because the "assignee" events are sent in series while nemobot tasks are run in parallel --> every task removes every assignee but the one added with that event --> every assignee is removed. Anyway, this is "ok" IMO as multi-assignees should be a no-op.

### tests not performed (domain coverage)
> At times not everything can be tested, and writing what hasn't been tested is just as important as writing what has been tested.

> An example of partial test is a field displaying 4 possible values. If 3 values are tested, with screenshots, and 1 is not, then it should be mentioned here.}
